src/session-summaries/the-intersection-observer-api.md

```diff
@@ -0,0 +1,19 @@
+## Intersection Observer Api
```

Comment:
Use a single `#`. That corresponds to an `h1` and that's how we'll be parsing it into HTML.